android/quest/src/main/assets/configs/app/profiles/household_profile_config.json

```diff
@@ -565,6 +565,7 @@
             "setOrganizationDetails": true,
             "type": "EDIT",
             "resourceIdentifier": "@{familyLogicalId}",
+            "resourceType": "Group",
```

Comment:
Do we need to include the questionnaire response action params to this config?

Reply:
1. no, household_profile_config.json doesn't need that.
2. resourceType as Group, is to fix launchContext issue. Where if not declared as, by default it tries to load a Patient, and in this case with a Group ID